libs/transactions/src/lib/views/transactions-view/transactions-view.component.html

```diff
@@ -5,13 +5,28 @@ <h1 *ngIf="title !== ''" data-role="transactions-view__title">
         {{ title }}
       </h1>
       <div class="bb-block bb-block--md">
-        <bb-text-filter (textChange)="filter = $event || ''"></bb-text-filter>
+        <bb-text-filter (textChange)="search($event)"></bb-text-filter>
+      </div>
+      <div
+        class="bb-block bb-block--md"
+        *ngIf="(bankAlias$ | async)?.['bankAlias'] as accountName"
+      >
+        Filtered By:
+        <bb-badge-ui
+          [text]="accountName"
+          color="success"
+          [abbr]="false"
+          [circle]="false"
+        ></bb-badge-ui>
+        <span routerLink="/transactions">X</span>
```

Comment:
In this way we also remove `search` query parameter :(

Reply:
<span routerLink="/transactions" [queryParams]="{search: filter}">X</span>

Is this okii?

Reply:
Perfect!